test/tests/val-2f/gold/experimental_data.csv

```diff
@@ -0,0 +1,132 @@
+temperature (C),flux (atoms/mm^2/s x 10^10)

```

Comment:
These are the values from the validation case you are starting from, please update this file to contain the value from the new validation case - and add other sets of experimental data if they are available for several damage rates.